content/articles/domain-access-control.md

```diff
@@ -29,56 +31,86 @@ With the Domain Access Control feature, you can set the level of access for each
 
 ## Setting access for a team member
 
-To set a team member's role for multiple domains at once, go to the Account > Members page. Click on the Manage Access button to configure the team member's access. You can give the team member [Full Access](#full-access) or [Limited Access](#limited-access).
+To set a team member's role for multiple domains at once:
+
+1. Go to your Account page.
+1. Click the **Members and seats** tab on the left side. On the Members card, you'll find each team member along with **Manage Access** and **Remove** buttons.
+1. Click **Manage Access** on a specific team member to configure their access level as [**Full Access**](#full-access) or [**Limited Access**](#limited-access).
 
-![Account Member's Page](/files/domain-access-control-members-page.png)
+![screenshot: set team member access](/files/team-member-access.png)
 
-### Full Access
+### Full access
 
-Any team member can be granted "Full Access". This gives them access to all resources inside an account – including all domains, contacts, certificates, templates, and billing.
+Any team member can be granted **Full Access**. This gives them access to all resources inside an account – including all domains, contacts, certificates, templates, and billing.
 
 ![Full Access](/files/domain-access-control-full-access.png)
 
-### Limited Access
+### Limited access
+
+You can specify **Limited Access** when inviting a team member. The new team member will be able to join and see the account, but cannot access any domains until you grant permission.
+
+![screenshot: invite a user with limited access](/files/invite-with-limited-access.png)
+
+If you invited someone with full access and want to change it, team members can be granted access to specific domains by setting their permission to **Limited Access**.
+
+1. Click **Manage access** for the specific team member.
+1. Select **Limited Access**.
+1. You'll see a list of each domain in your account, along with dropdown menus with options for [Zone Operator](#zone-operator) or [Domain Manager](#domain-manager).
 
-A team member can be granted access to specific domains by first setting their permission to "Limited Access". From there, you'll see a list of your account's domains, along with a control to set them as a [Zone Operator](#zone-operator) or [Domain Manager](#domain-manager) for each domain in the account. You can also enter part of a domain name, and you'll only see domains matching that search part, or you can enter a role name to see the domains where the team member has that role.
+![screenshot: set team member as limited access](/files/limited-access-team-member.png)
 
-![Limited Access](/files/domain-access-control-limited-access.png)
+4. You can search for specific domains, or enter a role to view domains for which that team member has been assigned a role.
```

Comment:
@dperrymorrow this is showing up as 1. instead of 4. in the netlify preview — can you help? I don't know how to fix that 😅

Reply:
whoops... let me fix that, I suspect there are some indentation problems in the markdown